execute_adhoc_plays.yml

```diff
@@ -7,7 +7,8 @@
 - name: Fetch adhoc playbooks
   get_url:
     url: "{{ playbook }}"
-    dest: "{{ config.defaults_dir + '/playbook.yml' }}"
+    dest: "{{ config.defaults_dir '/' + playbook|basename }}"
```

Comment:
Nice solution! You're missing a `+` after `config.defaults_dir`, should be
```
dest: "{{ config.defaults_dir + '/' + playbook|basename }}"
```

I was going to suggest something like
```
dest: "{{ config.defaults_dir + '/' + playbook|hash('sha1') + '-playbook.yml' }}"
```

... but your solution is probably better.